Dockerfile

```diff
@@ -1,14 +1,15 @@
-FROM ponylang/ponyc:release
+FROM ponylang/ponyc:release-alpine AS build
 
 WORKDIR /src/changelog-tool
 
 COPY Makefile LICENSE bundle.json /src/changelog-tool/
 COPY *.pony /src/changelog-tool/
 
 RUN make \
-  && make install \
-  && make clean
+ && make install
 
-WORKDIR /src/main
+FROM alpine:3.10
+
+COPY --from=build /usr/local/bin/changelog-tool /usr/local/bin/changelog-tool
 
 CMD changelog-tool
```

Comment:
Getting the following errors when running the image due to missing dependencies in the :

```
Error loading shared library libatomic.so.1: No such file or directory (needed by /usr/local/bin/changelog-tool)
Error loading shared library libgcc_s.so.1: No such file or directory (needed by /usr/local/bin/changelog-tool)
[...]
```

This is fixed by:

```diff
diff --git a/Dockerfile b/Dockerfile
index 7f8c695..15fc081 100644
--- a/Dockerfile
+++ b/Dockerfile
@@ -12,4 +12,6 @@ FROM alpine:3.10
 
 COPY --from=build /usr/local/bin/changelog-tool /usr/local/bin/changelog-tool
 
+RUN apk add --update libatomic gcc
+
 CMD changelog-tool
```

Reply:
addressed through a lot more work.

Reply:
This should be fixed via a lot more work.